README.md

```diff
@@ -39,9 +39,3 @@ class MyClass
 
 serialize(new MyClass); // will throw an exception
 ```
-
-The same applies to `DontDeserialise`, but this
-time with `unserialize()`.
-
-`Dont\JustDont` includes other seven traits and is the recommended one to use.
```

Comment:
I removed this bit as it lives on its own page, so it would be redundant to repeat it in the main README.